src/action.js

```diff
@@ -135,9 +134,7 @@ const showLintResults = async ([from, to]) => {
   )
   const formattedResults = formatErrors(lintedCommits)
 
-  // disable workflow commands
-  const token = uuidv4()
```

Comment:
The `uuidv4` dependency can be removed from this file, also as npm package if it is not used elsewhere.

Reply:
Nice catch! I see it's been even updated on https://github.com/wagoid/commitlint-github-action/pull/76, will merge that one. It's also about time to add eslint to this project, will add later on :smiley: